rom/dev/tests/test_fmcalias_derivation.rs

```diff
@@ -1,8 +1,14 @@
 // Licensed under the Apache-2.0 license
 
-use caliptra_builder::ImageOptions;
-use caliptra_hw_model::{Fuses, HwModel, ModelError};
+use caliptra_builder::{FwId, ImageOptions, APP_WITH_UART, ROM_WITH_UART};
+use caliptra_common::PcrLogEntry;
+use caliptra_common::PcrLogEntryId::*;
+use caliptra_hw_model::{BootParams, Fuses, HwModel, InitParams, ModelError, SecurityState};
+use caliptra_image_fake_keys::VENDOR_CONFIG_KEY_1;
+use caliptra_image_gen::ImageGenerator;
+use caliptra_image_openssl::OsslCrypto;
 use caliptra_image_types::IMAGE_BYTE_SIZE;
+use zerocopy::{AsBytes, FromBytes};
 
 pub mod helpers;
 
```

Comment:
[TODO] Use the error codes from the common library.
Can we resolve this as a part of this check in or you want to do it separately?

Reply:
Discussed offline; this TODO will be addressed with the common error crate changes.